packages/@okta/vuepress-site/conductor.yml

```
@@ -2523,7 +2523,78 @@ redirects:
     to: /docs/guides/set-up-self-service-registration/main/
   - from: /docs/guides/set-up-self-service-registration/next-steps/index.html
     to: /docs/guides/set-up-self-service-registration/main/
-
+  - from: /docs/guides/build-custom-ui-mobile/react-native/before-you-begin/
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/
+  - from: /docs/guides/build-custom-ui-mobile/react-native/create-an-okta-application/
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/
+  - from: /docs/guides/build-custom-ui-mobile/react-native/configure-packages/
+    to: /docs/guides/build-custom-ui-mobile/react-native/main/
```

```suggestion
    to: /docs/guides/build-custom-ui-mobile/react-native/main/#add-and-configure-packages
```
Not sure if you want to be specific. This is just a suggestion.